CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## v1.0.0
```

Comment:
@Madhu-1 lets call this as v1.0.0-pre.0. Rest looks good to me.

Reply:
@humblec done PTAL